docs/design.html

```diff
@@ -308,13 +308,12 @@ <h3><a id="replication" href="#replication">4.7 Replication</a></h3>
     In distributed systems terminology we only attempt to handle a "fail/recover" model of failures where nodes suddenly cease working and then later recover (perhaps without knowing that they have died). Kafka does not
     handle so-called "Byzantine" failures in which nodes produce arbitrary or malicious responses (perhaps due to bugs or foul play).
     <p>
-    We can now more precisely define that a message is considered committed when all in sync replicas for that partition have applied it to their log.
-    Only committed messages are ever given out to the consumer. This means that the consumer need not worry about potentially seeing a message that could be lost if the leader fails. Producers, on the other hand,
-    have the option of either waiting for the message to be committed or not, depending on their preference for tradeoff between latency and durability. This preference is controlled by the acks setting that the
-    producer uses.
-    Note that topics have a setting for the "minimum number" of in-sync replicas that is checked when the producer requests acknowledgment that a message
-    has been written to the full set of in-sync replicas. If a less stringent acknowledgement is requested by the producer, then the message can be committed, and consumed,
-    even if the number of in-sync replicas is lower than the minimum (e.g. it can be as low as just the leader).
+    We can now more precisely define that a message is considered committed when all in-sync replicas for that partition have applied the message to their log.
+    Only committed messages are ever given out to the consumer. This means that the consumer will not see a message that could be lost if the leader fails. Producers, on the other hand,
+    have the option of either waiting for the message to be committed or not, depending on their preference for tradeoff between latency and durability. This behavior is controlled by the producer acks setting.
```

Comment:
latency seems right to me, why do you suggest this change?

Reply:
No strong preference, just that it makes more sense to me to see two positive terms in a tradeoff. In my opinion, durability is a positive term but latency has a negative meaning to it.

Reply:
I agree that latency can have somewhat of a negative meaning to it but this doesn't seem correct to me. Throughput != Latency, right? You could have high throughput with bad latency (e.g large batching and compressing). The `acks` setting has a more direct relationship with latency rather than throughput in my opinion